openedx/stanford/djangoapps/register_cme/forms.py

```diff
@@ -71,7 +71,7 @@ class Meta(object):
                 'required': u'Please enter your Postal Code.',
             },
             'professional_designation': {
-                'required': u'Please select your Professional Designation.',
+                'required': u'Please select your Medical Designation or Professional Title.',
```

Comment:
If it is no longer Professional Designation then we should rename all instances in the code.